src/assets/images/Smilies.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Smilies = ({ className }) => (
```

Comment:
prop-types? aren't you getting a warning?

Reply:
Yes, I'm missing prop-types here. Do I have to configure ESLint to throw a warning ?

Reply:
yep you need `.eslintrc` file.